functions/Get-DbaJobOutputFile.ps1

```diff
@@ -0,0 +1,201 @@
+    The SQL Server that you're connecting to. Or an array of SQL Servers
+
+.PARAMETER SQLCredential
+    Credential object used to connect to the SQL Server as a different user be it Windows or SQL Server. Windows users are determiend by the existence of a backslash, so if you are intending to use an alternative Windows connection instead of a SQL login, ensure it contains a backslash.
+
+.PARAMETER JobName
+    The Agent Job Name to provide Output File Path for. Also available dynamically. If ommitted all Agent Jobs will be used
+
+.PARAMETER OpenFile
+    Uses Invoke-Item to open the file if it is available
+
+.EXAMPLE
+   Get-DbaJobOutputFile -SqlServer SERVERNAME -JobName 'The Agent Job' 
+
+   This will return the paths to the output files for each of the job step of the The Agent Job Job 
+   on the SERVERNAME instance  
+
+.EXAMPLE
+   Get-DbaJobOutputFile -SqlServer SERVERNAME 
+
+   This will return the paths to the output files for each of the job step of all the Agent Jobs
+   on the SERVERNAME instance   
+
+.EXAMPLE
+   Get-DbaJobOutputFile -SqlServer SERVERNAME -JobName 'The Agent Job' -OpenFile 
+
+   This will return the paths to the output files for each of the job step of the The Agent Job Job 
+   on the SERVERNAME instance and open the files if they are available
+
+.EXAMPLE
+   $Servers = 'SERVER','SERVER\INSTANCE1'
+   Get-DbaJobOutputFile -SqlServer $Servers -JobName 'The Agent Job' -OpenFile 
+
+   This will return the paths to the output files for each of the job step of the The Agent Job Job 
+   on the SERVER instance and the SERVER\INSTANCE1 and open the files if they are available
+
+.NOTES
+   AUTHOR - Rob Sewell https://sqldbawithabeard.com
+   DATE - 30/10/2016
+#>
+[CmdletBinding(SupportsShouldProcess = $true)]
+param
+(# The Server/instance 
+        [Parameter(Mandatory=$true,HelpMessage='The SQL Server Instance', 
+                   ValueFromPipeline=$true,
+                   ValueFromPipelineByPropertyName=$true, 
+                   ValueFromRemainingArguments=$false, 
+                   Position=0)]
+        [ValidateNotNull()]
+        [ValidateNotNullOrEmpty()]
+        [object]$SqlServer,
+        [Parameter(Mandatory=$false,HelpMessage='SQL Credential', 
+                   ValueFromPipeline=$true,
+                   ValueFromPipelineByPropertyName=$true, 
+                   ValueFromRemainingArguments=$false, 
+                   Position=1)]
+        [System.Management.Automation.PSCredential]$SqlCredential,
+        [Parameter(Mandatory=$false,HelpMessage='Want to open the file')] 
+        [switch]$OpenFile)
+    DynamicParam { if ($SqlServer) { return (Get-ParamSqlJobs -SqlServer $SqlServer -SqlCredential $SourceSqlCredential) } }
+
+	BEGIN 
+    {
+		$jobname = $psboundparameters.Jobs
+        # Create Table Object
+         $table = New-Object system.Data.DataTable $TableName 
+         # Create Columns
+         $col0 = New-Object system.Data.DataColumn Server,([string])
```

Comment:
hey rob, please use 

```
[pscustomobject]@{
ComputerName = $Server.ComputerNamePhysicalNetBIOS
InstanceName = $Server.Instancename
SqlInstance = $Server.Name
Job = $Step.Name
}
```

and so on. We do support datatables, but only when they are returned from the server. This release will have Out-DbaDataTable so it'll be easier.